onnxruntime/test/framework/execution_frame_test.cc

```diff
@@ -265,6 +265,8 @@ TEST_F(ExecutionFrameTest, MemPatternTest) {
   EXPECT_EQ(p->GetBlock(4)->offset_, 64);
 }
 
+// TODO: Uncomment after registering a kernel for opset 11 'ReduceSum' op
+/*
 TEST(ExecutionFrameTestWithoutSessionState, BadModelInvalidDimParamUsage) {
```

Comment:
see: https://github.com/google/googletest/blob/master/googletest/docs/faq.md#how-do-i-temporarily-disable-a-test

Reply:
Sure. Will do it. Thank you.

Reply:
Fixed.